.travis/tests.sh

```diff
@@ -16,8 +16,15 @@ then
         sudo python -m py_compile /home/cchq/www/travis/current/localsettings.py
     }
 
+    test_dimagi_environments() {
+        git clone https://github.com/dimagi/commcare-environments.git
+        ln -s commcare-environments/environments environments
```

Comment:
based on your comments in https://github.com/dimagi/commcare-environments/pull/3 I think you could just clone the repo to this location directly

Reply:
Oh yeah, thanks. Not sure how this test passed since commcare-environments/environments doesn't exist... Will investigate.

Reply:
Oh I see, since `environments/` still exists here, this link line is interpreted as "link commcare-environments/environments to environments/environments". I think for this PR I'm going to fix but then comment out these lines, since I'm not removing `environments/` from here just yet. That way the PR that removes environments will do less (hopefully just uncomment these lines), which will make the coordination easier.